grails-app/conf/application.yml

```diff
@@ -260,10 +260,10 @@ environments:
             passowrd: "password"
             logSql: false
 
-        elasticSearch:
-            client:
-                mode: local
-            bulkIndexOnStartup: true
+#        elasticSearch:
```

Comment:
I'd just delete this as it won't work anymore

Reply:
done